pkg/controllers/management/auth/project_cluster_handler_test.go

```diff
@@ -0,0 +1,54 @@
+package auth
+
+import (
+	"testing"
+
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/apis/management.cattle.io/v3/fakes"
+	"github.com/stretchr/testify/assert"
```

Comment:
newline unnecessary

Reply:
Just added to organize imports, github vs k8s